scripts/exregional_run_enkf.sh

```diff
@@ -376,7 +376,7 @@ use_gfs_nemsio=.true.,
 	lobsdiag_forenkf=.false.,
 	write_spread_diag=.false.,
 	netcdf_diag=${netcdf_diag:-.false.},
-        fv3_native=.true.,
+	fv3_native=.true.,
```

Comment:
Line 379 is still different between left and right. I guess you may use "tabs" while the left uses "spaces". I suggest you copy from the left and avoid making changes to this line.

Reply:
OK, I got it. the left one uses "8 spaces" instead of a "tab" as in Line 378. So your change to a tab is correct. Thanks!